### ISSUE TITLE
Does not save any input

### ISSUE THREAD

**Issue by martinstr** [TheDMSGroup/mautic-contact-client#248]
When we try to enter a URL in Send tab and then hit save it does not save the input.
No error in the log.

Tried on both Mautic 2.15.3 & 2.16.0

**Comment by heathdutton** [comment]
Do you have JS errors in your console? Try clicking out of the field before hitting apply.

**Comment by martinstr** [comment]
Yes, I have tested clicking out of the field before hitting apply 
Nope, sorry don't have any JS erros.

**Comment by martinstr** [comment]
Ok so I found out now that if you click on code tab and then enter the url there and click apply then it saves it. 
![image](https://user-images.githubusercontent.com/19844475/76780369-371ef400-67ad-11ea-98cc-02738c268bfa.png)
![image](https://user-images.githubusercontent.com/19844475/76780396-43a34c80-67ad-11ea-8d67-b86631e9eb57.png)

**Comment by MajesteitBart** [comment]
Same problem here, it saves it in the workaround you described, but its not sending any requests. Testing it with beeceptor. When I save it, it gives a 500 error. Seems like it has problems using composer, because of the dependency of dropbox/dropbox-sdk. That repo doenst exist anymore, and it stopped my complete install.

**Comment by gcoli6** [comment]
+1 same problem here. Mautic 2.16.2 | PHP 7.3.18

**Comment by geraldobenedito** [comment]
Anyone else could fix the error 500 at version 2.16.x ?

**Comment by geraldobenedito** [comment]
@MajesteitBart I'm facing this error at log file:

[2020-09-30 18:15:53] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ClassNotFoundException: "Attempted to load class "Mustache_Engine" from the global namespace. Did you forget a "use" statement?" at /var/www/html/plugins/MauticContactClientBundle/Helper/TokenHelper.php line 220 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ClassNotFoundException(code: 0): Attempted to load class \"Mustache_Engine\" from the global namespace.\nDid you forget a \"use\" statement? at /var/www/html/plugins/MauticContactClientBundle/Helper/TokenHelper.php:220)"} []

**Comment by insetic** [comment]
Hi, Anyone could resolve this issue?
mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ClassNotFoundException: "Attempted to load class "Mustache_Engine" from the global namespace. Did you forget a "use" statement?" at ...
